### DISCUSSION TITLE
RTX Video HDR support for video wallpapers

### DISCUSSION THREAD

**Discussion by vermi5** [rocksdanister/lively#2463] [Q&A] [answered]
Hello, this program is quite nice, I appreciate it being FOSS a lot.

I'm wondering two things about video wallpapers:

1- Why is `--aid` preferred over `stop-screensaver=no` to avoid display sleep timer/screensaver not triggering
2- Has anyone figured out how to enable RTX Video HDR for SDR video wallpapers? I replaced the built-in mpv binaries with newer ones where the feature is implemented but I'm simply unable to make it work even after fiddling with variations of `vf=d3d11vpp=nv12:nvidia-true-hdr` and other settings.

Thanks for reading.

**Comment by rocksdanister** [comment, marked as answer]
1. `stop-screensaver=no` is used already, its just for some systems `aid` workaround is required also.

**Comment by vermi5** [comment]
Closing this because the MPV developer who implemented RTX Video HDR has deleted his branch until he gets hardware to test or someone else debugs what's needed.
